docs/processes/qualityassurance/git-gontribution-guidelines.md

```diff
@@ -0,0 +1,224 @@
+
+The **footer** is optional. The Commit Message Footer format describes the purpose and structure of the footer.
+
+Any line of the commit message should be 100 characters or fewer.
+
+```text
+Commit Message Header
+<type>(<scope>): <short summary>
+  │       │             │
+  │       │             └─⫸ Summary in present tense. Not capitalized. No period at the end.
+  │       │
+  │       └─⫸ Commit Scope (optional): animations|common|style|forms|http|router|service-worker|
+  │                                     upgrade|changelog|dev-infra|docs-infra|migrations|
+  │
+  └─⫸ Commit Type: build|ci|docs|feat|fix|perf|refactor|test
+```
+
+The \<type\> and \<summary\> fields are mandatory, the (\<scope\>) field is optional.
+
+The \<type\> must be one of the following:
+
+- **build** : Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+- **ci** : Changes to our CI configuration files and scripts (example scopes: Circle, BrowserStack, SauceLabs)
+- **docs** : Documentation only changes
+- **feat** : A new feature
+- **fix** : A bug fix
+- **perf** : A code change that improves performance
+- **refactor** : A code change that neither fixes a bug nor adds a feature
+- **test** : Adding missing tests or correcting existing tests
+
+Doubtfire-LMS recommends Chris Beam's post on [How to Write Good Commit Messages](http://chris.beams.io/posts/git-commit/) to improve your commit message writing.
+
+**Use the imperative mood in your commit subject line**
+
+Write your commits in the imperative mood and not the indicative mood
+
+- "Fix a bug" and **not** "Fix*ed* a bug"
+- "Change the behaviour of Y" and **not** "_Changed_ the behaviour of Y"
+- "Add new API methods" and **not** "Sweet new API methods"
+
+Keep the subject line (top line) concise; keep it **within 50 characters**.
+
+Use the body (lines after the top line) to explain why and what and _not_ how; keep it **within 72 characters**.
+
+### But how can I write new lines if I'm using `git commit -m "Message"`?
+
+Don't use the `-m` switch. Use a text editor to write your commit message instead.
+
+If you are using the command line to write your commits, it is useful to set your git editor to make writing a commit body easier. You can use the following command to set your editor to Visual Studio Code, `nano`, `emacs`, `vim`, `atom`.
+
+```bash
+git config --global core.editor "code --wait"
+git config --global core.editor nano
+git config --global core.editor emacs
+git config --global core.editor vim
+git config --global core.editor "atom --wait"
+```
+
+# Git Workflow Summary
+
+## Start a new piece of work
+
+1. Synch repo and set up for new feature branch (remember to use Thoth Tech [branching guidelines](#__Branching_Guide) for naming the new branch):
+
+```bash
+git checkout main
+git pull main
+git checkout -b feature/\<project\_subtask\_description\>
```

Comment:
**suggestion:** The escaping characters look a bit off in the preview. Could we change it to use the example?

```sh
  git checkout -b feature/voice-verification-1.2-store-voice-input
```